test/REVLoansSourced.t.sol

```diff
@@ -501,7 +501,7 @@ contract REVLoansSourcedTests is TestBaseWorkflow, JBTest {
         // Ensure we actually received ETH from the borrow
         assertGt(USER.balance, 100e18 - 1e18);
 
-        // warp to after redemption rate is higher in the second ruleset
+        // warp to after cash out tax rate is higher in the second ruleset
```

Comment:
This comment is in quite a few tests but now that the `cashoutTaxRate` is reversed it seems like the second ruleset actually decreases the tax. Are the comments incorrect, or is there an issue somewhere?

Reply:
nice, done!